### DISCUSSION TITLE
React: autoprefixer: Replace color-adjust to print-color-adjust.

### DISCUSSION THREAD

**Discussion by mARTinKOPULETY** [twbs/bootstrap#36286] [Q&A]
Hi I try to make my first React app by some youtube tutorial. I have troubles with bootstrap. I install it, import it in the same way like tutorial do, but there  is the error autoprefixer: Replace color-adjust to print-color-adjust.. I found  here a  nice solution by @vineetdigit , but  Im pure  beginner and  I dont  understand what  author means with: "\rm -rf node_modules" and "restart the react app with "react-scripts start" " Thank you for your advice and your leniency.

This most certainly didn't work for me.

`npm install autoprefixer@10.4.5 --save-exact`

But following helped remove the warning.

In the package.json of the top level directory of the react app, add the following section after the dependencies: {..}

`"overrides":{
    "autoprefixer": "10.4.5"
  }`
delete package-lock.json
\rm -rf node_modules
npm install
restart the react app with "react-scripts start"
The warning is no more.

**Comment by dvselva** [comment]
the above solutions does not work. keep getting the warning. (6:29521) autoprefixer: Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.

**Comment by mdo** [comment]
https://github.com/twbs/bootstrap/issues/36259

**Reply by mARTinKOPULETY** [reply]
Thank you mdo.

**Comment by lucidlive** [comment]
`npm install bootstrap@5.2.0-beta1`

**Reply by dvselva** [reply]
Yes this took care of the issue: Thanks so much

**Reply by Suspend85** [reply]
Thanks. This solved my problem
